utils/wasm-proc/src/main.rs

```diff
@@ -36,6 +37,7 @@ impl std::fmt::Display for Error {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         match self {
             Self::OptimizerFailed => write!(f, "Optimizer failed"),
+            Error::WasmOptFailed => write!(f, "wasm-opt failed"),
```

Comment:
Need to either rename this entry or all remaining.
```suggestion
            Self::WasmOptFailed => write!(f, "wasm-opt failed"),
```